lib/Controller/Layout.php

```diff
@@ -2219,10 +2220,17 @@ public function export(Request $request, Response $response, $id)
             throw new InvalidArgumentException(__('Cannot manage tags on a Draft Layout'), 'layoutId');
         }
 
+        // Save As?
+        $saveAs = $this->getSanitizer()->getString('saveAs');
```

Comment:
That probably should be `$saveAs = $sanitizedParams->getString('saveAs')`

Reply:
ah yes, good catch thanks!